src/StockportWebapp/Views/stockportgov/Shared/ImageContentWidth.cshtml

```diff
@@ -1,10 +1,12 @@
 @model ContentBlock
 @if (Model is not null && !string.IsNullOrEmpty(Model.Image))
 {
+    bool hasBody = !string.IsNullOrEmpty(Model.Body);
+
     <section class="content-block content-block--margin image-banner image-banner-content-width" data-cy="image-content-width">
         <div class="grid-container">
             <div class="grid-100">
-                <figure>
+                <figure aria-label="@(hasBody ? $"{Model.Title} image" : string.Empty)">
```

Comment:
Should the aria-label not be alt text from the model rather than the title?

Also, should the aria-label be on the image rather than the figure?